src/main/java/usecases/CourseEvaluator/CheckAnswer.java

```diff
@@ -0,0 +1,24 @@
+package usecases.CourseEvaluator;
+import entities.Course;
+
+
+public class CheckAnswer {
+
+    public int compare(Course student, Course original) {
+
+        int counter = 0;
+
+        for (int i = 0; i < student.getQuestions().size(); i++){
+            if ((student.getQuestionAt(i).getAnswer()).equalsIgnoreCase
+                    ((original.getQuestionAt(i).getAnswer()))){
+                counter += original.getQuestionAt(i).getPoints();
```

Comment:
Thank you for implementing my suggestion on removing the case when comparing the answers !!